exporters/prometheus/exporter.go

```diff
@@ -110,6 +111,7 @@ func New(opts ...Option) (*Exporter, error) {
 		withoutCounterSuffixes: cfg.withoutCounterSuffixes,
 		disableScopeInfo:       cfg.disableScopeInfo,
 		scopeInfos:             make(map[instrumentation.Scope]prometheus.Metric),
+		scopeInfoErrs:          make(map[instrumentation.Scope]error),
```

Comment:
We do not need to store the whole error. It is unused. How about something like:

```suggestion
		scopeInvalid:          make(map[instrumentation.Scope]struct{}),
```